### PR TITLE
Closes #2: add migration for Item

### LINKED ISSUE #2: 2. Visitor can view items
Background: I have several items and each of them has a title, description, price, and image

As a visitor When I visit “/items” I can see all existing items

### PR DESCRIPTION
**Pull request by dphilla**
# Issue Number: 2

# Description of changes:

Sets up migration and model for Item

# @mentions of the person or team responsible for reviewing proposed changes:
@liambarstad @mikeyduece @Aram-Anderson

### CODE REVIEW

File: db/migrate/20170912213251_create_items.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateItems < ActiveRecord::Migration[5.1]
```
Comment (mikeheft) [inline db/migrate/20170912213251_create_items.rb:1]: Best Migration I've ever seen, very well done. Huzzah!!!

File: app/models/item.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class Item < ApplicationRecord
+end
```
Comment (Aram-Anderson) [inline app/models/item.rb:2]: I like this, Dan. Good use of ruby/rails conventions. A+

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  create_table "items", force: :cascade do |t|
+    t.float "price"
+    t.integer "stock_level"
+    t.string "image"
+    t.string "name"
+    t.text "description"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```
Comment (Aram-Anderson) [inline db/schema.rb:27]: 👍

File: db/migrate/20170912213251_create_items.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  def change
+    create_table :items do |t|
+      t.float :price
+      t.integer :stock_level
+      t.string :image
```
Comment (liambarstad) [inline db/migrate/20170912213251_create_items.rb:6]: 💯 💯 💯

**Comment by mikeheft** [comment]
@dphilla Looks good to me, never in my life have I ever seen such a robust schema! 💯 🥇

**Pull request merged**